### PR TITLE
CA-484

### PR DESCRIPTION
**Pull request by dominicusplatus**

### CODE REVIEW

File: CollAction/Views/Manage/Index.cshtml [hunk 1]
```diff
@@ -9,95 +9,152 @@
     ViewData["Title"] = "Manage your account";
 }
 
+<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (ljwboschker) [inline CollAction/Views/Manage/Index.cshtml]: I think jquery and bootstrap styling are already present in the current codebase. Can you check and see what's already there to use?

Comment (logophobia) [reply CollAction/Views/Manage/Index.cshtml]: Additionally, if you load from different sources, content security policy will start blocking things.

Comment (dominicusplatus) [reply CollAction/Views/Manage/Index.cshtml]: Yes I know sorry this is WIP, was supposed to remove it ;)

File: CollAction/Views/Manage/Index.cshtml [hunk 2]
```diff
@@ -9,95 +9,152 @@
...
+            <div class="container">
+                <div class="panel panel-default">
+                        <div class="panel-heading"><dt>Newsletter Subscription:</dt></div>
+                        <div class="panel-body">
+                            <div class="row">
+                                <div class="col-md-12">
+                                        <form asp-controller="Manage" asp-action="NewsletterSubscription" method="post" class="form-horizontal">
+                                            <div asp-validation-summary="All" class="text-danger"></div>
+                                            <label asp-for="NewsletterSubscription" class="control-label"></label>
+                                            @Html.CheckBoxFor(m => m.NewsletterSubscription, new { onclick = "this.form.submit();"})
```
Comment (logophobia) [inline CollAction/Views/Manage/Index.cshtml]: This doesn't work anymore with CSP, see: https://github.com/CollActionteam/CollAction/pull/279

**Pull request closed without merging**